src/join/join.rs

```diff
@@ -0,0 +1,374 @@
+#![crate_name = "uu_join"]
+
+/*
+ * This file is part of the uutils coreutils package.
+ *
+ * (c) Konstantin Pospelov <kupospelov@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+extern crate getopts;
```

Comment:
I don't, generally. However, `clap` by default changes how the `--help` output looks like. I briefly looked through its documentation and how it's used in `cp`, and it seems that the `-h` and `-V` option descriptions which are auto-generated by clap start with a capital letter, while previously all the option descriptions were in lower case. To be consistent, either the auto-generated options should be overridden by ones with custom lower case messages, or the first letter of all the other options should be also capitalized. Is it OK to go with the first approach? I'm actually asking because it is not consistent for `cp`.

Reply:
Yeah just override the auto-generated ones.  That seems to be the best solution to me.